tv_apps/smart-system/test/marionette/app_install_dialog_test.js

```diff
@@ -70,6 +66,8 @@ marionette.skip('app install manager tests', function() {
   test('install app and cancel it', { devices: ['tv'] }, function() {
     // install app
     appInstaller.install(serverManifestURL);
+    system.waitForEvent('modal-dialog-opened');
+
```

Comment:
Wait for modal dialog opened so make sure the operations below go under modal dialog opened.